config/prometheus.yaml

```diff
@@ -160,3 +187,102 @@ spec:
     - protocol: TCP
       port: 9090
       targetPort: 9090
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: alertmanager-config
+  namespace: monitoring
+data:
+  config.yml: |-
+    global:
+    templates:
+      - '/etc/alertmanager/*.tmpl'
+    route:
+      receiver: default
+      group_by: [alertname]
+      group_wait: 30s
+      group_interval: 5m
+      repeat_interval: 2h
+      routes:
+    receivers:
+      - name: 'default'
+        sns_configs:
+        - topic_arn: $PROMETHEUS_ALERTS_ARN
+          subject: '[Alert Manager $CLIENT_ID] {{ .GroupLabels.alertname }}'
+          sigv4:
+            region: $AWS_DEFAULT_REGION
+            access_key: $AWS_ACCESS_KEY_ID
```

Comment:
https://prometheus.io/docs/alerting/latest/configuration/#sigv4_config

We should set up an IAM role and use that role instead of access keys.